src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -54,6 +54,7 @@
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
   <ItemGroup>
+  <PackageReference Include="Autodesk.IDSDK" Version="1.1.5" />
```

Comment:
Can you remind me why do we need the dependency here as well as DynamoCore?

Reply:
we don't need it here, I can remove this

Reply:
@zeusongit  Autodesk.IDSDK is library private not public to open sources ?

Reply:
yes, it is private for now.